airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -199,3 +203,8 @@ class UserLocationReport(IncrementalGoogleAdsStream):
     UserLocationReport stream: https://developers.google.com/google-ads/api/fields/v8/user_location_view
     Google Ads API field mapping: https://developers.google.com/google-ads/api/docs/migration/mapping#geo_performance
     """
+
+
+class ClickView(IncrementalGoogleAdsStream):
```

Comment:
Please add the doc-string with URL info and other context information to help future devs :)

Reply:
resolve